### PR TITLE
Elm decode

### PR DESCRIPTION
**Pull request by andrewMacmurray**
+ set up elm decoders to receive data from phoenix api
+ http request to process and put quotes, services and weightings into model as `Dict`s
+ draft functions to navigate through the list of quotes

#59 #27

### CODE REVIEW

File: web/elm/Data/Web.elm [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    decode RawWeighting
+        |> required "quote_id" int
+        |> required "service_id" int
+        |> required "weight" float
+
+
+getQuoteIdsFromWeightings : List RawWeighting -> List Int
+getQuoteIdsFromWeightings rawWeightings =
+    let
+        filterUnique =
```
Comment (ivanmauricio) [inline web/elm/Data/Web.elm]: Should this be `removeDuplicates`?

File: web/elm/Data/Web.elm [hunk 2]
```diff
@@ -0,0 +1,93 @@
+module Data.Web exposing (..)
```
Comment (ivanmauricio) [inline web/elm/Data/Web.elm]: Are we still calling this module `Data.Web`?

File: web/elm/Data/Web.elm (hunk 1 above)
Comment (andrewMacmurray) [inline web/elm/Data/Web.elm]: that's a great idea!

File: web/elm/Data/Web.elm (hunk 2 above)
Comment (andrewMacmurray) [inline web/elm/Data/Web.elm]: maybe `Api`?

Comment (ivanmauricio) [inline web/elm/Data/Web.elm]: that sounds like a good idea :neckbeard:

**Pull request merged**